src/OpenStack.Net/OpenStack/Services/ObjectStorage/V1/AccountMetadataExtensions.cs

```diff
@@ -0,0 +1,291 @@
+﻿namespace OpenStack.Services.ObjectStorage.V1
+{
+    using System;
+    using System.Collections.Immutable;
+
+    /// <summary>
+    /// This class provides extension methods for obtaining specific account information from the account metadata
+    /// provided by the service.
+    /// </summary>
+    /// <seealso cref="GetAccountMetadataApiCall"/>
+    /// <seealso cref="IObjectStorageService.PrepareGetAccountMetadataAsync"/>
+    /// <seealso href="http://docs.openstack.org/api/openstack-object-storage/1.0/content/GET_showAccountDetails__v1__account__storage_account_services.html">Show account details and list containers (OpenStack Object Storage API V1 Reference)</seealso>
+    /// <threadsafety static="true" instance="false"/>
+    /// <preliminary/>
+    public static class AccountMetadataExtensions
+    {
+        /// <summary>
+        /// Gets the name of the <c>X-Account-Bytes-Used</c> HTTP header, which the Object Storage Service may
+        /// include with the <see cref="AccountMetadata"/> to specify the total size of all objects stored in the
+        /// Object Storage Service for the account.
+        /// </summary>
+        /// <seealso cref="GetBytesUsed(AccountMetadata)"/>
+        public const string AccountBytesUsed = "X-Account-Bytes-Used";
```

Comment:
These fields should probably be `static readonly` unless there is a particular requirement to have them `const` (e.g. using them as an argument to an attribute).